### PR TITLE
Fix Windows XP support

### LINKED ISSUE #11300: Linux: use of win32 structure outside of win32 ifdef
In file Core/FileSystems/DirectoryFileSystem.cpp
Line 903, structure findInfo declared inside ifdef win32, but used outside of it, thus linux build failes.

### PR DESCRIPTION
**Pull request by unknownbrackets**
I think this should work well enough for XP.

Leaking a single kernel32.dll reference, which I think is fine.

Also fixes #11300.

-[Unknown]

### CODE REVIEW

File: Core/FileSystems/DirectoryFileSystem.cpp [hunk 1]
```diff
@@ -900,7 +900,7 @@ std::vector<PSPFileInfo> DirectoryFileSystem::GetDirListing(std::string path) {
 		localtime_r((time_t*)&s.st_atime,&entry.atime);
 		localtime_r((time_t*)&s.st_ctime,&entry.ctime);
 		localtime_r((time_t*)&s.st_mtime,&entry.mtime);
-		if (!listingRoot || (wcscmp(findData.cFileName, L"..") && wcscmp(findData.cFileName, L".")))
+		if (!listingRoot || (wcscmp(dirp->d_name, L"..") && wcscmp(dirp->d_name, L".")))
```
Comment (weihuoya) [inline Core/FileSystems/DirectoryFileSystem.cpp]: `dirp->d_name` is `char` array, no matching function for call to `wcscmp`

Comment (unknownbrackets) [reply Core/FileSystems/DirectoryFileSystem.cpp]: Derp.  That's what I get for copy/pasting.  I changed it afterward (I had a big if around the whole thing), and decided this change was better, then copy/pasted it both places.

-[Unknown]

**Comment by sarami55** [comment]
PPSSPPWindows.exe( 32bit) on "Automated PPSSPP builds site" still says "entry point not found GetFinalPathNameByHandleW".

**Comment by mrcmunir** [comment]
GetFinalPathNameByHandleW Not exist on oficial kernel XP kernel32.dll (but Will working on Extended kernel patched for XP).

**Comment by sarami55** [comment]
GetFinalPathNameByHandleW(static value) is declared
in MERGE file "Io: Fallback to long/full path on Windows XP".
I think that error will  not  happen.
i wonder if compiling is something wrong.

**Pull request merged**